packages/orchestrator-ui-components/src/components/WfoSubscriptionsList/WfoSubscriptionsList.tsx

```diff
@@ -118,6 +118,16 @@ export const WfoSubscriptionsList: FC<WfoSubscriptionsListProps> = ({
             field: 'note',
             name: t('note'),
         },
+        customerFullname: {
```

Comment:
Do they have to be at the most right side of the table -- position in this config object determines the location of the column

Reply:
Something for the acceptance feedback